lib/modulesync.rb

```diff
@@ -14,7 +14,7 @@ def self.local_file(config_path, file)
   end
 
   def self.module_file(puppet_module, file)
-    "#{PROJ_ROOT}/#{puppet_module}/#{file}"
+    "#{puppet_module}/#{file}"
```

Comment:
Could just add project_root as another argument and avoid having to do the string concatenation later